commit d9138c78c3e8cb286864509fc444ebb4484c3d70
Author: Neil Horman <nhorman@tuxdriver.com>
Date:   Mon Jan 27 06:58:34 2014 -0500

    Adjust default hintpolicy value
    
    affinity_hint values are something of a holdover from prior to when irqbalance
    was re-written.  Previous to the rewrite irqbalance could not determine with
    great accuracy the device to which an msi(x) irq was associatied, the node it
    was local to, etc, and so was not able to balance it well.  kernel affinity_hint
    values were created to work around that by just telling user space where to put
    an interrupt.  However, since the rewrite, irqbalance is perfectly capable of
    parsing all information about an irq out of sysfs, and can make superior policy
    decisions about balancing according to user input, over kernel suggestions.  As
    such, allow users to honor affinity_hint, but by default ignore it
    
    Signed-off-by: Neil Horman <nhorman@tuxdriver.com>

diff --git a/irqbalance.1 b/irqbalance.1
index ff156c7..2610cfa 100644
--- a/irqbalance.1
+++ b/irqbalance.1
@@ -57,6 +57,8 @@ irq is balanced, but the assigned object will be a subset of the affintiy hint
 .P
 .I ignore
 irq affinity hint value is completely ignored
+.P
+The default value for hintpolicy is ignore
 
 .TP
 .B -p, --powerthresh=<threshold>
diff --git a/irqbalance.c b/irqbalance.c
index c697d47..2b79eac 100644
--- a/irqbalance.c
+++ b/irqbalance.c
@@ -48,7 +48,7 @@ int foreground_mode;
 int numa_avail;
 int need_rescan;
 unsigned int log_mask = TO_ALL;
-enum hp_e hint_policy = HINT_POLICY_SUBSET;
+enum hp_e hint_policy = HINT_POLICY_IGNORE;
 unsigned long power_thresh = ULONG_MAX;
 unsigned long deepest_cache = ULONG_MAX;
 unsigned long long cycle_count = 0;
